sorts/radix_sort.py

```diff
@@ -43,6 +43,53 @@ def radix_sort(list_of_ints: list[int]) -> list[int]:
         placement *= RADIX
     return list_of_ints
 
+def radix_with_counting_sort(list_of_ints: list[int]) -> List[int]:
```

Comment:
```suggestion

def radix_with_counting_sort(list_of_ints: list[int]) -> list[int]:
```
- PEP 8: 2 lines of whitespace between functions
- Please use `list` rather than `List` for type annotations